documentation/source/user_section/tutorials/spinalcord-segmentation/sct_deepseg_sc-example-t1.rst

```diff
@@ -0,0 +1,32 @@
+Hands-on Example: T1
+####################
+
+.. figure:: https://raw.githubusercontent.com/spinalcordtoolbox/doc-figures/master/spinalcord_segmentation/t1_deepseg_before_after.png
+   :align: right
+   :figwidth: 20%
```

Comment:
here also i would move the figure to a separate line, after the command, and make it side by side